nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/jdbc/JDBCRowMapper.java

```diff
@@ -1077,8 +1077,8 @@ protected Serializable copyHierRecursive(IdWithTypes source, Serializable parent
         }
         // recurse in children
         boolean onlyComplex = parentId == null;
```

Comment:
Here please add a comment explaining that if we create a version (`parentId == null`) then we exclude regular children, in the case of versionable folderish.